### DISCUSSION TITLE
Curating a GitHub topic on JSON Schema

### DISCUSSION THREAD

**Discussion by egekorkan** [json-schema-org/community#506] [Vision and direction]
Currently, different JSON Schema projects use a different topic in their GitHub that results in two views on GitHub:

1. https://github.com/topics/jsonschema
2. https://github.com/topics/json-schema

Neither of them is curated, meaning that nothing shows up in the introduction part. An example of Kotlin (it was the first available one) can be seen [here](https://github.com/topics/kotlin). I think we should do two things:

1. Agree on the topic to use and hope that others pick up on the one that with some promotion.
2. Curate that topic with a very simple readme, like the one at the beginning of https://github.com/json-schema-org (a logo and one sentence). It is mainly creating a PR at https://github.com/github/explore/blob/main/CONTRIBUTING.md .

**Comment by benjagm** [comment]
This is a great idea @egekorkan!! I'll say we curate the two tags.

I am adding this topic to the agenda of the next Open Community Working Meeting and hopefully someone in the community can start working as soon as possible.

Again, this is a small task with huge impact... Thanks for the suggestion!

**Reply by egekorkan** [reply]
I think having a page in both but nudging people to the preferred one would be the best way. Otherwise projects will start using both "to be sure"

**Reply by benjagm** [reply]
@egekorkan is it ok if I send the PR adding you are reviewer?

**Reply by benjagm** [reply]
I can't add you, but this is the PR : https://github.com/github/explore/pull/4020

Please let me know what do you think. Happy to add missing information or improve the content.

**Reply by egekorkan** [reply]
Looks mostly good. I have left a small review comment:)

**Reply by benjagm** [reply]
Thanks! I saw the comment and did a new push to provide the full list of authors of all the spec versions. 

Thanks a lot for this and have a great weekend!

**Comment by benjagm** [comment]
The PR has been merged. Thanks @egekorkan.

**Comment by Relequestual** [comment]
Did we determine if we could get one topic to redirect to the other?

**Reply by egekorkan** [reply]
So this turned out to be more annoying than I thought. The two pages are definitely linked but it is not a strict redirecting, like what I am used to from redirecting a webpage resource. Here are my observations:

- The title "JSON Schema" is only applied to the main one. The other one has a title which is just the topic name, i.e. "jsonschema"
- The part below the title and list of repositories is the same (see screenshot at the end) but there is one exception:
  - Followers: If I star only one of the topics, the follower count is increased in both but one is marked as starred. If I star both, follower count still increases by one.
- Since it is not a redirect, the repositories listed are different. The main one starts with `FastAPI` whereas the other one starts with `python-jsonschema`.

I am not sure what parts are bugs and what are features.

![image](https://github.com/json-schema-org/community/assets/20195407/414e2235-6fa9-4b9a-b3ad-a12c5bc08942)

**Reply by Relequestual** [reply]
Appreciate you digging into this. Honestly, what I'd suggest is to reach out to GitHub support. It's a little hard to find sometimes but they are actually really responsive and helpful. I usually have pretty tricky tickets. Set your expectations high. (of course, if you have time and don't mind that is.)

**Reply by benjagm** [reply]
I sent a new PR to add more details to the Topic like the repo link but I don't think we should add details to recommend one primary Topic over another. The goal of the Topics is to help other people find and contribute to our project but not to measure or count. I couldn't find a single Topic providing guidance to use one primary topic over other secondary topic, so I think we should not do it.
